flake.nix

```diff
@@ -475,7 +475,11 @@
               checkPhase = ''
                 cd $src/.
                 for i in `find . -name "*.nix" -type f`; do
-                    nil diagnostics "$i"
+                    if [[ $i =~ 'vendor' ]]; then
+                      echo "skipping $i"
+                    else
+                      nil diagnostics "$i"
+                    fi
```

Comment:
why not do this in the src filter?

Reply:
Good point

Reply:
8d4fbb2dd26f3b8ffd4bf6ea40973e794f6a0446